sdk/analytics/src/test/java/com/klaviyo/analytics/networking/KlaviyoApiClientTest.kt

```diff
@@ -100,14 +116,12 @@ internal class KlaviyoApiClientTest : BaseTest() {
 
     @Test
     fun `Enqueues a profile API call`() {
```

Comment:
Are you adding additional assertions to existing tests for logs. Kind of a nit but maybe worth isolating to different tests. Like if something changed with the logging and a test called might be confusing.

Reply:
I did a bit of both

Reply:
But fair point, i can probably test all the logging i need in just 1 added test rather than adding to each of these

Reply:
I put logging into its own tests and tested for the actual output.